Documentation/network/kubernetes/ciliumendpointslice.rst

```diff
@@ -6,28 +6,112 @@
 
 .. _CiliumEndpointSlice:
 
-******************
```

Comment:
I consolidated `Documentation/network/kubernetes/ciliumendpointslice.rst` with `Documentation/network/kubernetes/ciliumendpointslice_beta.rst` since they had redundant content and it makes more sense to have just one single page dedicated to CES